### DISCUSSION TITLE
(bigger) dep updates

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#2533] [Ideas] [answered]
Continuing #1396:

To continue this, here is a list of (big) dep updates (even if some already have Issues, I want to mention them), I will remove updates from this list when they are done:
- alpine v3.18 (waiting for v3.18.1): #2768 
   * watchtower
   * domaincheck
   * borg
   * apache/httpd
   * nextcloud
   * mastercontainer
   * imaginary (install and final image)
   * talk container 4.6.2-r1-alpine (alpine 3.18) #1992 
- Elastic v8 (ubuntu/debian based?) #2332
- PHP 8.2 Nextcloud #2534

**Comment by Zoey2936** [comment]
If I read https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.18.0, I don't see there big changes that could break something

**Comment by szaimen** [comment]
> * alpine v3.18 (waiting for v3.18.1?) — watchtower, domaincheck, borg, apache

yes, lets wait for 3.18.1

> * imaginary, alpine 3.18 for the final and go image build part

lets wait for 3.18.1

> * will the Nextcloud container be updates to alpine v3.17 or v3.18?

With the update to 26 we can directly upgrade alpine to 3.18 if 3.18.1 is out by then

> * will the Nextcloud container be updated next to php 8.1 or 8.2?

8.1. See https://github.com/nextcloud/all-in-one/issues/1782 and https://github.com/nextcloud/all-in-one/issues/2534

> * php 8.2? mastercontainer

See https://github.com/nextcloud/all-in-one/pull/2536

> * alpine v3.18 mastercontainer

lets wait for 3.18.1

> * talk container 4.6.2-r1-alpine (alpine 3.18)

lets wait for 3.18.1

> * Elastic v8 (ubuntu/debian based?)

See https://github.com/nextcloud/all-in-one/issues/2332

> * apache/httpd alpine 3.18

lets wati for 3.18.1

> * (postgres has no alpine verison pinning)
> * (clamav has no alpine verison pinning)
> * (redis has no alpine verison pinning)
> * office excluded from this list (ubuntu/debian based?)
> * copying of docker/caddy/watchtower/spread binaries has no alpine version pinning
> * nats binary has no OS at all

the above is fine

**Reply by Zoey2936** [reply]
I've updated the list

**Reply by szaimen** [reply]
Great, thanks! :)

**Reply by Zoey2936** [reply]
But one question, will you update the nextcloud image to alpine 3.18 with 3.18.1 or php8.2/nc27+?

**Reply by szaimen** [reply]
> nextcloud image to alpine 3.18 with 3.18.1 

that one :)

**Reply by Zoey2936** [reply]
https://gitlab.alpinelinux.org/alpine/aports/-/issues/14995

**Reply by Zoey2936** [reply]
Alpine v3.18.2 was published (yes 3.18.2 not 3.18.1—it seems the tag was created (without publishing files) and then they made a hotfix to an alpine versioning package and released 3.18.2)

**Comment by szaimen** [comment, marked as answer]
Will be done in https://github.com/nextcloud/all-in-one/pull/2768. The rest will be done in linked issues.
